app/gen-server/migration/1751350685797-ServiceAccounts.ts

```diff
@@ -77,6 +77,7 @@ export class ServiceAccounts1751350685797 implements MigrationInterface {
         onDelete: 'CASCADE',
       })
     );
+    // Useful?
```

Comment:
Not useful for the tuple `(id, owner_id)`.

Through we should create an index for both `(owner_id)` and `(service_user_id)`.